documentation/API_documentation/QoSProfile_Mapping_Table.md

```diff
@@ -1,11 +1,25 @@
 # QoS Profiles Mapping Table (REFERENCE DRAFT)
 
-| Profiles | User Info | 5Qi mapping |
+Here are two possible examples of mapping tables from QoS Profiles to provided connectivity characteristics. The second example describes a case, where the network is specially prepared in well define geographical areas for supporting Quality on Demand needs.
+
+Example 1: Considering wide-area network usage.
+| QOS Profile labels | Network Service Description | Network Service Constraints |
+|----------| --------- | ----------- |
+| QOS\_E   | Latency stays stable under congestion (throughput upto 2Mbps). The application bitrate should not exceed 2Mbps. | The service is only provided, when the UE is located within the geographical area, as agreed in the Service Level Agreement.<br>The access to the network service is restricted to a maximal number of simultaneous session. |
```

Comment:
```suggestion
| QOS\_E   | Latency stays stable under congestion (throughput upto 500kbps). The application bitrate should not exceed 500kbps. | The service is only provided, when the UE is located within the geographical area, as agreed in the Service Level Agreement.<br>The access to the network service is restricted to a maximal number of simultaneous session. |
```
The intention of this QoS profile is to support use case which need a reliable connectivity even under congestion but with a low bit rate.

The sentence "The access to the network service is restricted to a maximal number of simultaneous session." fits better to the QoS profiles with higher bandwidth (e.g. QoS_L).

"The service is only provided, when the UE is located within the geographical area, as agreed in the Service Level Agreement." ... can be read differently (geographical region within a network? A region containing several networks?). In addition suggesting to delete pointers to SLAs.

Reply:
It is certainly ok to lower the bitrates. Still, some admission control might still be needed, even when lowering the bitrates.  Thus, adding this sentence "The access to the network service is restricted to a maximal number of simultaneous session."  to QoS_L is ok. However, not sure about removing it from QOS_E.

The definition of the geographical region is intentionally a bit blurry. One geographical region can be "the home network" or even larger. When deleting the SLA, does this mean that there is no SLA or such geographical area definition is not mentioned in the SLA?

Reply:
The pull request should include these comments now. Please check.